TESTS/mbed_hal/lp_us_tickers/main.cpp

```diff
@@ -444,7 +444,7 @@ Case cases[] = {
     Case("hf ticker overflow test", hf_ticker_setup, ticker_overflow_test),
     Case("hf ticker increment test", hf_ticker_setup, ticker_increment_test),
```

Comment:
Hi

Maybe not the good PR... but maybe we should rename hf ticker into us ticker here ?

Reply:
I'll do it in a separate PR. I need this one ASAP to get docs generated .